pkg/networkservice/mechanisms/kernel/kernelvethpair/server.go

```diff
@@ -22,48 +22,61 @@ package kernelvethpair
 import (
 	"context"
 	"fmt"
+	"net/url"
 
 	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/networkservicemesh/api/pkg/api/networkservice/mechanisms/kernel"
+	"github.com/pkg/errors"
 
 	"github.com/networkservicemesh/api/pkg/api/networkservice"
 
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
 
+	"github.com/networkservicemesh/sdk-vppagent/pkg/networkservice/vppagent"
 	"github.com/networkservicemesh/sdk-vppagent/pkg/tools/kernelctx"
-	"github.com/networkservicemesh/sdk-vppagent/pkg/tools/netnsinode"
 )
 
-type kernelVethPairServer struct {
-	fileNameFromInodeNumberFunc func(string) (string, error)
-}
+type kernelVethPairServer struct{}
 
 // NewServer provides NetworkServiceServer chain elements that support the kernel Mechanism using veth pairs
 func NewServer() networkservice.NetworkServiceServer {
-	return &kernelVethPairServer{fileNameFromInodeNumberFunc: netnsinode.LinuxNetNSFileName}
+	return &kernelVethPairServer{}
 }
 
 // NewTestableServer - same as NewServer, but allows provision of fileNameFromInodeNumberFunc to allow for testing
 func NewTestableServer(fileNameFromInodeNumberFunc func(string) (string, error)) networkservice.NetworkServiceServer {
```

Comment:
fileNameFromInodeNumberFunc is unused, do we really need NewTestableServer?

Reply:
You are correct, that's an artifact of a previous approach that should have been trimmed from the code. Removed.